### ISSUE TITLE
openload: Unable to extract stream URL (test URL itself!)

### ISSUE THREAD

**Issue by zopieux** [rg3/youtube-dl#16246]
*This is a duplicate of #16137, that was locked for unknown reason, with a “cant-reproduce” tag even though the **test URL itself does not work**.*

---

- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.04.16**

I have
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

This is a 
- [x] Bug report (encountered problems with youtube-dl)

---

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://openload.co/f/kUEfGclsU9o']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.04.16
[debug] Python version 3.6.5 (CPython) - Linux-4.15.15-1-ARCH-x86_64-with-arch
[debug] exe versions: ffmpeg 3.4.2, ffprobe 3.4.2, phantomjs 2.1.1, rtmpdump 2.4
[debug] Proxy map: {}
[Openload] kUEfGclsU9o: Downloading embed webpage
[Openload] kUEfGclsU9o: Executing JS on webpage
ERROR: Unable to extract stream URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 789, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/common.py", line 440, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/openload.py", line 344, in _real_extract
    'stream URL'))
  File "/usr/lib/python3.6/site-packages/youtube_dl/extractor/common.py", line 808, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract stream URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

---

### Description of your *issue*, suggested solution and other information

The [test URL](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/openload.py#L249) itself does load in the browser, but doesn't in youtube-dl latest version.

**Comment by Zopieux** [comment]
Is this `cant-reproduce` tag a joke?

**Comment by dstftw** [comment]
No jokes here. It's locked due to spam and senseless comments. If you want something to be done on the issue you must provide reproducible environment.

**Comment by Zopieux** [comment]
@dstftw The reproductibility issue is that your network (and Travis CI's) is not IPv6-enabled (or at lease openload doesn't want to deliver you IPv6 content), while @phoenixhaxor and mine is. Fixed in #16250, thanks for reviewing.
